### ISSUE TITLE
About environment installation issues.

### ISSUE THREAD

**Issue by moral66** [zjzser/TraceableSpeech#1]
There is no requirements.txt file in the folder.

**Comment by zjzser** [comment]
Thx, now I add it

**Comment by moral66** [comment]
Thank you for your reply. Additionally, I would like to ask how the LibriTTS dataset is split. Thanks again for taking the time to respond to my question.

**Comment by zjzser** [comment]
well, I remember the training set consists of train-clean-100, train-clean-360, and train-other-500. but I think it‘s a flexiable things.

**Comment by moral66** [comment]
Hello, I would like to ask why the model trained for 150,000 epochs shows an accuracy of only over 50% when tested using inference.py. Is there something wrong with my settings?
I used train-clean-100 as the training set and dev-clean as the validation set, and I took audio from a specific folder in the dev-clean for testing. Since validation took a long time during training, I adjusted the validation interval to 10,000. Could the validation interval be an issue?

config.json
`
"batch_size": 8,
"num_workers": 1,
`

inference.py
` 
parser.add_argument('--input_wavs_dir', default='/LibriTTS_wav/test/8230/279154')
parser.add_argument('--output_dir', default='./4-16-at/test_output_wav/libiritts')
 parser.add_argument('--checkpoint_file', default='./save_model/g_00150000') # python inference.py --checkpoint_file ./save_model/g_00150000
`

train.py
`   
parser.add_argument('--input_mels_dir', default=None)
parser.add_argument('--input_training_file', default='LibriTTS_wav/train') 
parser.add_argument('--input_validation_file', default='LibriTTS_wav/val')
parser.add_argument('--checkpoint_path', default='./save_model')
parser.add_argument('--config', default='./config.json')
parser.add_argument('--training_epochs', default=2000, type=int)
parser.add_argument('--stdout_interval', default=5, type=int) # 5
parser.add_argument('--checkpoint_interval', default=5000, type=int) # 20 5000
parser.add_argument('--summary_interval', default=100, type=int) # 100 # $ tensorboard --logdir /your_path/ --port 8080
parser.add_argument('--validation_interval', default=10000, type=int) # 20 1000 500
parser.add_argument('--num_ckpt_keep', default=5, type=int) # 5
parser.add_argument('--fine_tuning', default=False, type=bool)
`

![Image](https://github.com/user-attachments/assets/4aa61645-bdbf-468a-96c2-56d5bb90ea7c)

**Comment by zjzser** [comment]
well, Firstly, i don‘t think validation interval is the reason. I have two opinons
1: maybe you can try the biggest train set, and monitor the tensorboard message. the 50% means that the model only extract two digits. Furthermore, 150000 is the updated times, not epoch.
2: maybe you can try a bigger test set to mean it…but in the case you shown, I guess it don’t work.
